docs/collections.md

```diff
@@ -13,24 +13,24 @@ Then run the following command:
 The script will ask you to run additional commands if it ran successfully:
 
 1. Start added containers using docker-compose  
-    ```shell
-    $ docker-compose up -d
-    ```
+```shell
+$ docker-compose up -d
+```
```

Comment:
This indentation is too small and it breaks the list. Four spaces keeps the code block within the `<li>`.